tests/patch.go

```diff
@@ -0,0 +1,39 @@
+// From https://gist.github.com/imosquera/6716490#sthash.O4z2aQQp.LUHz2Cbb.dpuf
```

Comment:
I'm seeing a lot of copying (not just this project).  Instead, please use github.com/heketi/tests.  I do not mind moving to another repo in the future, but in the mean time, it is better to pull from the same area.